pages/project/[id].js

```diff
@@ -62,3 +70,12 @@ const TaskItem = styled.li`
   font-size: 15;
   overflow: hidden;
 `;
+
+const DeleteButton = styled.button`
+  border: none;
+  font-weight: 600;
+  border-radius: 50px;
+  font-size: 13px;
+  height: 25px;
+  width: 30px;
+`;
```

Comment:
You repeat stylings for li, ul, h1 on several pages: Maybe you want to outsource this in a component. 
When properties differ you can use variant-syntax with css in the styledComponents.

Reply:
Apart from this small notes I made, it looks good and your delete-function works properly!